examples/platforms/gp712/entropy.c

```diff
@@ -41,22 +41,22 @@
 
 #include "utils/code_utils.h"
 
-#if __SANITIZE_ADDRESS__ != 0
+#if defined(__SANITIZE_ADDRESS__) && __SANITIZE_ADDRESS__ != 0
```

Comment:
Is this change necessary? Also, do you use the address sanitizer on this platform? If not, we could just remove these defines.